packages/core/strapi/src/commands/index.ts

```diff
@@ -62,6 +63,12 @@ const strapiCommands = {
   uninstallCommand,
   versionCommand,
   watchAdminCommand,
+  /**
+   * Cloud
+   */
+  loginCommand: cloudCli.login.command,
```

Comment:
Any reason not to register with with buildStrapiCloudCommand ?

Reply:
`buildStrapiCloudCommand` was built to create a standalone CLI for cloud commands (mainly with testing purposes in mind).
I can adapt to have a buildStrapiCloudCommand that loads all the cloud commands inside the given commander in parameter 🤔